paasta_itests/steps/chronos_steps.py

```diff
@@ -67,15 +69,22 @@ def list_chronos_jobs_has_job(context, should_or_not):
         assert context.chronos_job_name in job_names
 
 
-# NOTE this is a placeholder until we are able to get per-job task information from Chronos
-@then(u"the {old_or_new_job} {has_or_not} running tasks")
-def chronos_check_running_tasks(context, old_or_new_job, has_or_not):
-    # job_id = context.old_chronos_job_name if old_or_new_job == 'old job' else context.chronos_job_name
-    # if has_or_not == "has no":
-    #     assert job_id has no running tasks
-    # else:  # has_or_not should be "has"
-    #     assert job_id has running tasks
-    assert True
+@then(u'the job stored as "{job_name}" {has_or_not} running tasks')
```

Comment:
What if this was just 

```
"the {job_name} job is {status}"
```

And then you can just assert that "myjob" is "idle", etc?